x-pack/plugins/observability_solution/dataset_quality/common/constants.ts

```diff
@@ -19,3 +19,10 @@ export const NONE = 'none';
 
 export const DEFAULT_TIME_RANGE = { from: 'now-24h', to: 'now' };
 export const DEFAULT_DEGRADED_DOCS = { percentage: 0, count: 0 };
+
+export const SO_SEARCH_LIMIT = 10000;
```

Comment:
Isn't this too much? is this the the number of items we could fit in the panel?
I don't think any user would go through this amount of items to find a dashboard, could we do something like bringing the first `X items` and then a link to dashboard page filtering by the integration?

Reply:
> I would highly doubt any integration would have this many dashboard assets not even close to it

Then why to specify this big number here? I don't have an specific example, just thinking about the usability of a list of 10k items. 

> How would you want to prioritize the first `X items` to be displayed

I would show the first `X items` in the way they are returned from the SO client, but maybe this is a question more close to UX